native/cocos/audio/android/AudioPlayerProvider.cpp

```diff
@@ -445,5 +449,44 @@ void AudioPlayerProvider::resume() {
         _pcmAudioService->resume();
     }
 }
+bool AudioPlayerProvider::registerPcmData(const ccstd::string &audioFilePath, PcmData &data) {
+    std::lock_guard<std::mutex> lck(_pcmCacheMutex);
+    if (_pcmCache.find(audioFilePath) != _pcmCache.end()){
+        CC_LOG_DEBUG("file %s pcm data is already cached.", audioFilePath.c_str());
+        return true;
+    }
+    _pcmCache.insert(std::make_pair(audioFilePath, data));
+    return true;
+}
 
+bool AudioPlayerProvider::getHeader(const ccstd::string &audioFilePath, PCMHeader &header) {
+    _pcmCacheMutex.lock();
```

Comment:
Use `std::lock_guard<std::mutex> lck(_pcmCacheMutex);` instead of `lock unlock` manually.

Reply:
1